vm/adb/adb.go

```diff
@@ -156,7 +156,15 @@ func (pool *Pool) Create(workdir string, index int) (vmimpl.Instance, error) {
 		log.Logf(0, "associating adb device %v with console cmd `%v`", inst.device, inst.consoleCmd)
 	} else {
 		if inst.console == "" {
+			// More verbose log level is required, otherwise echo to /dev/kmsg won't show.
+			level, err := os.ReadFile("/proc/sys/kernel/printk")
```

Comment:
`os.ReadFile` will look up the printk value on the host process. You need to use `inst.adb()` to query the value on the VM/device.